html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/Authentication/Login.pm

```diff
@@ -62,10 +88,24 @@ Execute this module
 sub execute_child {
     my ($self) = @_;
     if($self->app->request->method eq "POST"){
-        $self->authenticate();
+        if($self->app->request->path eq "challenge") {
```

Comment:
One detail.. I don't see that path defined in lib/pf/web/constants.pm
To my knowledge, it won't work without it in the Apache captive portal